### PR TITLE
Translate Jun-3 and Mid-1

### PR DESCRIPTION
**Pull request by remind-pro**
test changes

### CODE REVIEW

File: frontend/junior-3/css.md [hunk 1]
```diff
@@ -1,59 +1,58 @@
...
+* What is progressive jpeg?
+* When to use the 'img' tag, and when 'background-image'?
+  * What is DPI and PPI
+  * What does the phrase "screen with high pixel density" mean?
+  * What is 'device-pixel-ratio'?
+  * How to scale graphics on screens with high pixel density (for example, on retina displays)?
+
+## General issues
+
+* How does 'calc()' work? What parameters does it take? What operations are allowed? What units can be used for setting parameter values?
```
Comment (Znack) [inline frontend/junior-3/css.md:37]: Здесь и в других местах надо обратить вгимание на кавычки, там не просто одинарные в оригинале, а именно обратные. Это нужно, чтобы текст форматировался :)

Comment (Znack) [reply frontend/junior-3/css.md:37]: image

Вот тут можно посмотреть, как увидеть отформатированный текст :)

File: frontend/junior-3/git.md [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
+    * What is the unsafety of using rebase?
+  * What do these commands do, how to use them, and when are they useful:
     * `stash`
     * `reflog`
     * `bisect`
-* Как отменить merge commit?
-* Что такое `HEAD` и `detached HEAD`? Что значат `HEAD^`, `HEAD~`, `HEAD@{1}`?
-* Что такое Git flow, какие знаете примеры?
+* How to cancel?
+* What are `HEAD` and `detached HEAD`? ЧWhat do `HEAD^`, `HEAD~`, `HEAD@{1}` mean?
```
Comment (Znack) [inline frontend/junior-3/git.md:25]: Ч? :)

Comment (remind-pro) [reply frontend/junior-3/git.md]: Исправила))

File: frontend/junior-3/css.md (hunk 1 above)
Comment (remind-pro) [reply frontend/junior-3/css.md:37]: Хорошо, поняла, исправлю везде.

File: frontend/junior-3/testing.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-* Как тесты помогают при рефакторинге?
-* В чём преимущества правила одной концепции на один unit-тест?
-* В чём преимущества изолированных друг от друга тестов?
-* Как найти баланс между количеством тестов и покрытием кода? Какие проблемы могут быть из-за излишнего количества юнит-тестов?
+* What are boundary conditions? Why to test them?
+* What is a heisenbug? What are the ways of debugging it?
+* What is fixture?
+* What is code coverage? What tools are there to analyze it?
+* How do unit tests differ from integration tests? Why are integration tests more difficult to write and maintain?
+* Why to pay attention to patterns when tests fail? How can an analysis that shows which combination of tests fell off help?
```
Comment (stanislav-az) [inline frontend/junior-3/testing.md:8]: >Why to pay attention to patterns when tests fail?

Здесь не очень понятно что имеются ввиду именно закономерности (patterns) отказов, а не какие-то другие.

File: frontend/junior-3/testing.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+* How do tests help in design? Why is the module's testability an excellent indicator of "the code purity"?
+* How do tests help with refactoring?
+* What are the advantages of the one concept rule per one unit test?
+* What are the advantages of separate tests?
```
Comment (stanislav-az) [inline frontend/junior-3/testing.md:12]: Здесь не понятно что тесты не должны перекрываться по своему смыслу, или тестируемой функции, слишком общо

File: frontend/junior-3/testing.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
-# Тестирование
+# Testing
```
Comment (stanislav-az) [inline frontend/junior-3/testing.md:1]: Можно сразу перенести раздел в общие вопросы после ревью. Думаю нужно создать на это таску?

Comment (remind-pro) [reply frontend/junior-3/testing.md:1]: Раздел в общие вопросы перенести? Или все вопросы вынести в таск?

Comment (stanislav-az) [reply frontend/junior-3/testing.md:1]: Это мы уже после перевода думаю будем делать. Хотелось бы чтобы разделы про тестирование оказались когда-нибудь в `/shared` @Znack

Comment (Znack) [reply frontend/junior-3/testing.md:1]: Да я думаю, это стоит сделать отдельной задачей и мы тут уже сами справимся :) То есть в этом пулл реквесте ничего править в этом плане не надо :)

File: frontend/middle-1/isomorphism.md [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
-  * Какие есть составляющие структуры для каждой архитектуры?
-  * Что обычно делает клиентский сервер при изоморфной архитектуре?
-  * Как в каждом случае осуществляется раутинг?
-  * Какие зоны ответственности у front-server, API-server и непосредственно самого клиента? Какие вычислительные нагрузки ложатся на каждую из этих частей?
-* Как из обычного приложения сделать изоморфное? (подробно рассказать про изоморфность, посмотреть примеры реализации)
-* Можем ли мы добиться изоморфного SPA, не поднимая SSR-сервер?
-* Как можно осущестить pre-fetching данных в изоморфном приложении?
-* Когда вы бы посоветовали заказчику использовать клиентский рендер, а когда изоморфный?
+* What is server/client/isomorphic rendering?
+* What are the advantaged and drawbacks of each of rendering types?
```
Comment (RostislavaFSD) [inline frontend/middle-1/isomorphism.md:4]: advantageS

File: frontend/middle-1/isomorphism.md [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
+* Describe the general architecture of server/client/isomorphic rendering:
+  * What are the components of each architecture?
+  * What does a client server usually perform provided that the architecture is isomorphic?
```
Comment (RostislavaFSD) [inline frontend/middle-1/isomorphism.md:7]: What a client server usually does perform

Comment (chmnkh) [reply frontend/middle-1/isomorphism.md:7]: не соглы, насколько я помню does в начале должен стоять, потому что это вопрос
